Bug Reporting/ARKEmailBugReporter.m

```diff
@@ -40,6 +40,7 @@ @interface ARKEmailBugReporter () <MFMailComposeViewControllerDelegate, UIAlertV
 
```

Comment:
Changes to this file are the meat of the fix. The removing of Crashlytics from the sample app was ancillary